test/PushNotificationTests/TestFunctions.h

```diff
@@ -0,0 +1,18 @@
+﻿class PushNotificationTestFunctions
```

Comment:
You don't really need a class to wrap those functions. They can just be under a namespace.

Reply:
Makes sense since these are all statics anyway